python/rikai/types/video.py

```diff
@@ -59,42 +58,53 @@ def __init__(self, vid: str):
     def __repr__(self) -> str:
         return "YouTubeVideo({0})".format(self.vid)
 
-    def _repr_html_(self):
-        from IPython.lib.display import YouTubeVideo
+    def show(self, **kwargs):
+        """
+        Visualization in jupyter notebook with custom options
+
+        Parameters
+        ----------
+        kwargs: dict
+            Key-word args for IPython.display.YouTubeVideo to show in
+            Jupyter notebook
 
-        return YouTubeVideo(self.vid)._repr_html_()
+        Returns
+        -------
+        v: IPython.display.YouTubeVideo
+        """
+        from IPython.display import YouTubeVideo
+
+        return YouTubeVideo(self.vid, **kwargs)
+
+    def _repr_html_(self):
+        """default visualization in jupyter notebook cell"""
+        return self.show()._repr_html_()
 
     def __eq__(self, other) -> bool:
         return isinstance(other, YouTubeVideo) and self.vid == other.vid
 
-    def get_stream(self, download: bool = False, **kwargs) -> "VideoStream":
+    def get_stream(self, ext="mp4", quality="worst") -> "VideoStream":
         """
-        Get one specific stream of this youtube video
+        Get a reference to a particular stream
 
         Parameters
         ----------
-        download: bool, default False
-            By default just return a remote https reference. If True then
-            actually download the data
-        kwargs
+        ext: str, default 'mp4'
+            The preferred extension type to get
+        quality: str, default 'worst'
+            Either 'worst' or 'best'
```

Comment:
worth to use an `Enum`?

Reply:
probably not rn. Let's let the API stablize first